compiler/src/parsing/lexer.mll

```diff
@@ -177,7 +177,7 @@ rule token = parse
   | "fail" { FAIL }
   | "exception" { EXCEPTION }
   | "try" { TRY }
-  | "raise" { RAISE }
+  | "throw" { THROW }
```

Comment:
Does this lexer change make this a breaking? (I forget if this PR includes a breaking note)

Reply:
Mmm, yeah it does (if you used `throw` as an identifier).